internal/functions/deploy/deploy.go

```diff
@@ -94,6 +94,7 @@ func Run(ctx context.Context, slug string, projectRefArg string, noVerifyJWT *bo
 			}
 		}
 
+		buildScriptPath := filepath.Join(scriptDirPath, "build.ts")
```

Comment:
Seems a little error prone to assume that the caller knows what files are inside `scriptDirPath`. Shall we make `CopyDenoScripts` return a struct with explicit path fields?

```go
type ScriptDir struct {
	ExtractPath string
	BuildPath   string
}
```

Reply:
updated in https://github.com/supabase/cli/pull/794/commits/5c95cb9d57de7d137ec88e1b9e07240e9b0a9c20